nevergrad/optimization/optimizerlib.py

```diff
@@ -2539,6 +2539,9 @@ def _num_objectives_set_callback(self) -> None:
                 # override at runtime
                 self._optim = DE(self.parametrization, self.budget, self.num_workers)
 
+    def recommend(self) -> p.Parameter:
+        return super().recommend()
```

Comment:
We do not want to use an algorithm-specific recommendation, it is often not that good...

Reply:
This modifies NGOpt8 and NGO. I would say it's worth it -- we are already different from any existing publication, so it's good to do the right thing rather than to focus on stability. I would rather focus on stabilization on Dec. 31 and then do a new variant.